providers/route53/route53Provider.go

```diff
@@ -305,3 +385,31 @@ func (r *route53Provider) EnsureDomainExists(domain string) error {
 	return err
 
 }
+
+func (r *route53Provider) waitUntilNameserversUpdate(operationId *string) error {
```

Comment:
This is related to the desired UX for dnscontrol.

Updating nameservers in route53domains returns an operationid immediately, but doesn't complete for ~30-45 seconds. Is your expectation that DNS control would just see that the request completed successfully, or is it okay to wait until we have confirmation that the change was successfully made?